### DISCUSSION TITLE
GCP app engine [Aug 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3362] [Discord Threads]
> 🙋‍♂️ **Philipp Tsipman**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### I think with cloud SQL some socket param's have to be set  
#  
> 💁 **o1lab**  
#### pls see if you can get a sample node app working with knex before  
#  
> 💁 **o1lab**  
#### then it can be worked backwards  
#  
> 🙋‍♂️ **Philipp Tsipman**  
#### Thank you!  I'm closer.    
####    
#### Google App Engine connects to SQL via Unix Sockets rather than TCP. So, I switched to a Knex JSON file format to write out the full host.  Like this:   
####    
#### app.yaml   
#### env_variables:   
####   NC_DB_JSON_FILE: './dbconnect.json'   
####    
#### dbconnect.json   
#### {   
####     "client": "pg",   
####     "connection": {   
####         "host" : "/cloudsql/project:region:instance",   
####         "user" : "my_user",   
####         "password" : "my_password",   
####         "database" : "db_name"   
####     }   
#### }   
#### (Source: https://cloud.google.com/sql/docs/postgres/manage-connections)   
####    
#### Now, I can log in and can see my NocoDB project and tables. Yay.   
####    
#### However, none of the table data is showing. Every table times out.    
####    
#### To be continued...  
#  
> 🙋‍♂️ **Philipp Tsipman**  
#### I can see the table names and table column names. But, the data is not showing:   
####    
#### "A :: KnexTimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?"  
#  
> 🙋‍♂️ **Philipp Tsipman**  
#### Any advice?  
#  
> 💁 **o1lab**  
#### Are u doing from gae or cloud run  
#  
> 💁 **o1lab**  
#### With cloud run thr were issues related to pool  
#  
> 💁 **o1lab**  
#### However I cannot see anything specific from our end  
#  
> 💁 **o1lab**  
#### Could you also try setting pool connection limits  
#  
> 💁 **o1lab**  
#### In your config  
#  
> 💁 **o1lab**  
#### Pls lookup knex  
#  
> 💁 **o1lab**  
#### For it  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1008864848517550081)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
